gapic-generator/templates/default/service/rest/service_stub/grpc_transcoding_method/_def.erb

```diff
@@ -1,24 +1,32 @@
 <%- assert_locals method -%>
 ##
+# @private
+#
 # GRPC transcoding helper method for the <%= method.name %> REST call
 #
 # @param request_pb [<%= method.request_type %>]
 #   A request object representing the call parameters. Required.
 # @return [Array(String, [String, nil], Hash{String => String})] 
 #   Uri, Body, Query string parameters
 def <%= method.rest.transcoding_helper_name %> request_pb
-  uri = "<%= method.rest.uri_interpolated %>"
-  <%- if method.rest.body? -%>
-  body = <%= method.rest.body_interpolated %>
-  <%- else -%>
-  body = nil
-  <%- end -%>
-  query_string_params = {}
-  <%- if method.rest.query_string_params? -%>
-  <%- method.rest.query_string_params.each do |field| -%>
-  <%= render partial: "service/rest/service_stub/grpc_transcoding_method/def/query_string_param", locals: { field: field } -%>
-  <%- end -%>
-  <%- end -%>
-
-  [uri, body, query_string_params]
+  transcoder = Gapic::Rest::GrpcTranscoder.new
+    .with_bindings(
```

Comment:
See how the final generated code (several files down) indents this. Best to indent it that way here in the template rather than depend on rubocop to fix it.

Reply:
:+1: